_episodes/41-code-review.md

```diff
@@ -245,106 +176,8 @@ The following diagram depicts the branches that you should have in the repositor
 Adapted from <a href="https://sillevl.gitbooks.io/git/content/collaboration/workflows/gitflow/" target="_blank">Git Tutorial by sillevl</a> (Creative Commons Attribution 4.0 International License)
 </p>
 
-To achieve this, the following steps are needed.
-
-#### Step 1: Adding Collaborators to a Shared Repository
-
-You need to add the other team member(s) as collaborator(s) on your repository
-to enable them to create branches and pull requests.
-To do so, each repository owner needs to:
-
-1. Head over to Settings section of your software project's repository in GitHub.
-   ![Accessing settings for a repository in GitHub](../fig/github-settings.png){: .image-with-shadow width="900px"}
-2. Select the **vertical** tab 'Collaborators' from the left and click the 'Add people' button.
-   ![Managing access to a repository in GitHub](../fig/github-manage-access.png){: .image-with-shadow width="900px"}
-3. Add your collaborator(s) by their GitHub username(s), full name(s) or email address(es).
-   ![Adding collaborators to a repository in GitHub](../fig/github-add-collaborators.png){: .image-with-shadow width="900px"}
-4. Collaborator(s) will be notified of your invitation to join your repository
-   based on their notification preferences.
-5. Once they accept the invitation, they will have the collaborator-level access to your repository
-   and will show up in the list of your collaborators.
-
-See the full details on
-[collaborator permissions for personal repositories](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository)
-to understand what collaborators will be able to do within your repository.
-Note that repositories owned by an organisation have a
-[more granular access control](https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github)
-compared to that of personal repositories.
```

Comment:
I restructured the examples so they don't require adding collaborators. I would be tempted to move this into the extras though, as it might be useful in some instances to know how to do this.